skimage/data/__init__.pyi

```diff
@@ -32,6 +32,7 @@ __all__ = [
     'moon',
     'nickel_solidification',
     'page',
+    'palisades_of_vogt',
```

Comment:
Is this a change necessary on main?

Reply:
I noticed that this was missing when I previously added a method here (which I've later reverted). This change is out of scope for this PR; will remove.